test/Interpreter/SDK/Inputs/OldABI/OldABI.mm

```diff
@@ -158,13 +158,13 @@ static void initialize(void) {
 
 bool CanTestOldABI() {
   // These tests don't work until the stable ABI is using its designed bit.
-  // This check can be removed after SWIFT_DARWIN_ENABLE_STABLE_ABI_BIT
-  // is set everywhere.
   Class cls = objc_getClass("_TtCs19__EmptyArrayStorage");
   if (!cls) abort();
   uintptr_t *words = (uintptr_t *)cls;
   if ((words[4] & 3) != 2) return false;  // wrong stable is-Swift bit
   return true;
+  // This check can be removed after SWIFT_CLASS_IS_SWIFT_MASK is made
+  // static everywhere.
```

Comment:
What check is this referring to?

Reply:
I messed up the change here, and the comment is supposed to replace the removed comment above it and refer to the "wrong stable is-Swift bit" check.